includes/definitions/discovery/liebert.yaml

```diff
@@ -60,8 +66,22 @@ modules:
                     oid: lgpPduPsTable
                     value: lgpPduPsEntryApTotal
                     num_oid: '.1.3.6.1.4.1.476.1.42.3.8.30.20.1.90.{{ $index }}'
-                    descr: 'Apparent Power for {{ $lgpPduPsEntryModel }}'
+                    descr: 'ApPwr {{ $lgpPduPsEntryModel }}'
                     index: 'lgpPduPsEntryApTotal.{{ $index }}'
+                    group: ApparentPower
+                -
+                    oid: lgpPduRcpTable
+                    value: lgpPduRcpEntryPwrOut
+                    num_oid: '.1.3.6.1.4.1.476.1.42.3.8.50.20.1.65.{{ $index }}'
+                    descr: 'Pwr {{ $lgpPduRcpEntryUsrLabel }}'
```

Comment:
Power?

Reply:
yes - same issue as above. Adding long prefixes hides the actual device name